src/main/java/com/google/finapp/SpannerDao.java

```diff
@@ -22,7 +22,7 @@
 
 import java.math.BigDecimal;
 
-final class SpannerDao {
+final class SpannerDao implements com.google.finapp.SpannerDaoInterface {
```

Comment:
You don't need to fully qualify the class. Just implements SpannerDaoInterface should suffice.